src/components/Wave/Waves.tsx

```diff
@@ -60,15 +66,15 @@ const Waves = ({ parentRef }: { parentRef?: MutableRefObject<HTMLDivElement | nu
   })
 
   return (
-    <CanvasContainer style={{ height: currentParentSize.height }}>
-      <canvas id="canvas" ref={canvasRef} width={currentParentSize.width} height={currentParentSize.height} />
+    <CanvasContainer style={{ height: `${size.current.height}px` }}>
+      <canvas id="canvas" ref={canvasRef} width={size.current.width} height={size.current.height} />
     </CanvasContainer>
   )
 }
 
 const waves: [WaveEntity, WaveEntity] = [
-  new WaveEntity([0.0081, 0.028, 0.015], 1, 1, ['rgba(22, 204, 244, 0.6)', 'rgba(101, 16, 248, 0.2)']),
-  new WaveEntity([0.0022, 0.018, 0.005], 1, 1.2, ['rgba(244, 129, 22, 0.6)', 'rgba(101, 16, 248, 0.2)'])
+  new WaveEntity([0.0081, 0.028, 0.015], 1, 1, ['rgba(22, 204, 244, 0.6)', 'rgba(101, 16, 248, 0)']),
+  new WaveEntity([0.0022, 0.018, 0.005], 1, 1.2, ['rgba(244, 129, 22, 0.6)', 'rgba(101, 16, 248, 0)'])
```

Comment:
Why the opacity change? 
Setting it back to 0.2, otherwise the contrast at the bottom of the box is not good enough.

Reply:
This is just an artifact of copying over that code. I noticed it too but figured it wasn't significant...!